core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1340,34 +1338,49 @@ public long getNewBlockIdForFile(AlluxioURI path) throws FileDoesNotExistExcepti
 
   @Override
   public Map<String, MountPointInfo> getMountTable() {
-    SortedMap<String, MountPointInfo> mountPoints = new TreeMap<>();
-    for (Map.Entry<String, MountInfo> mountPoint : mMountTable.getMountTable().entrySet()) {
-      MountInfo mountInfo = mountPoint.getValue();
-      MountPointInfo info = mountInfo.toMountPointInfo();
-      try (CloseableResource<UnderFileSystem> ufsResource =
-          mUfsManager.get(mountInfo.getMountId()).acquireUfsResource()) {
-        UnderFileSystem ufs = ufsResource.get();
-        info.setUfsType(ufs.getUnderFSType());
-        try {
-          info.setUfsCapacityBytes(
-              ufs.getSpace(info.getUfsUri(), UnderFileSystem.SpaceType.SPACE_TOTAL));
-        } catch (IOException e) {
-          // pass
-        }
-        try {
-          info.setUfsUsedBytes(
-              ufs.getSpace(info.getUfsUri(), UnderFileSystem.SpaceType.SPACE_USED));
-        } catch (IOException e) {
-          // pass
-        }
-      } catch (UnavailableException | NotFoundException e) {
-        // We should never reach here
-        LOG.error("No UFS cached for {}", info, e);
-        continue;
+    return mMountTable.getMountTable().entrySet().stream()
```

Comment:
since defaultfsmaster is not threadsafe, getMountInfo can still throw an InvalidPathException. What happens to Collectors.toMap if an exception is thrown?

Reply:
The `mMountTable.getMountTable` will return a copy of the `MountTable` and `getMountPointInfo(MountInfo)` will not throw `InvalidPathException`. It just transform the `MountInfo` to `MountPointInfo` and add the ufs space information.

For safety consideration, I will change the lambda expression back to normal for loop.